python/src/com/jetbrains/python/PyGotoSymbolContributor.java

```diff
@@ -45,6 +42,7 @@
     symbols.addAll(PyModuleNameIndex.getAllKeys(project));
     symbols.addAll(StubIndex.getInstance().getAllKeys(PyFunctionNameIndex.KEY, project));
     symbols.addAll(StubIndex.getInstance().getAllKeys(PyVariableNameIndex.KEY, project));
+    symbols.addAll(StubIndex.getInstance().getAllKeys(PyClassAttributesIndex.KEY, project));
```

Comment:
`getAllKeys(PyClassAttributesIndex.KEY, project)` already fetches the names of all the attributes of all classes, including instance attributes and methods. Only those elements that match the pattern in `getItemsByName()` are used for creating a navigation item. So in the worst case we'll create navigation items for all the instance attributes that contain a single letter.

Instance attributes are cached in a class in `myInstanceAttributes` so it's not that time-consuming to find an attribute by its name.